application/F3DOptionsParser.h

```diff
@@ -31,6 +31,12 @@ struct F3DAppOptions
   std::vector<int> Resolution{ 1000, 600 };
   bool Quiet = false;
   bool Verbose = false;
+  double CameraAzimuthAngle = 0.0;
+  double CameraElevationAngle = 0.0;
+  std::vector<double> CameraFocalPoint;
```

Comment:
no default value like other options?

Reply:
no, if not set, it is computed automatically. that was already the case before.

Reply:
(it is intialized to an empty vector, which we rely on to know that it was not set)